sdk-libs/macros/docs/features/anchor-account-features.md

```
@@ -0,0 +1,376 @@
+    pub system_program: Program<'info, System>,
+}
+```
+
+---
+
+## 1. `init`
+
+**Purpose**: Creates a new account via CPI to the System Program.
+
+**Behavior**:
+- Allocates space on-chain
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)